test/mocks/mock-schedule.js

```diff
@@ -13,6 +13,7 @@ const baseSchedule = {
   email: 'farmer@farms.com',
   documentReference,
   address,
+  remainingAmount: 200.01,
```

Comment:
We should use our mock component variable here

Reply:
Already mocked, mock reference added